decentraland/unity-renderer
# chore: remove avatars lods feature flag

## What does this PR change?

Removed the avatars lod feature flag. Also removed the enabled bool attribute, since it was redundant without the flag's presence. Moved the flag initialization code into the Initialize method.

Some tests started failing since they were dependent on the enabled bool. Modified these tests accordingly to the modified code.

## How to test the changes?

Go to: https://play.decentraland.zone/?renderer-branch=chore/remove-avatars-lods-feature-flag
Go to WonderMine scene
Check that the avatar lods are present when the number of avatars exceeds the Max HQ Avatars setting

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
